Bot/TynibotHost.cs

```diff
@@ -24,12 +23,21 @@ public class TynibotHost
 
         private DefaultHandler DefaultHandler = null;
         private readonly Dictionary<string, IChannelHandler> ChannelHandlers = new Dictionary<string, IChannelHandler>();
-        private readonly Dictionary<string, SlashCommand> SlashCommands = new Dictionary<string, SlashCommand>()
+
+        private readonly List<SlashCommand> SlashCommands = new List<SlashCommand>()
```

Comment:
Refactored this so that the name of the command comes from the command metadata.

Reply:
Nice I like that as well